data/card_catalogue/social.json

```diff
@@ -27,27 +27,27 @@
   },
   {
     "title": "Arouse",
-    "upgrade": "Seduce",
+    "upgrade": "Bond",
     "tier": "red",
     "details": {
       "rules": {
         "active": "Causes a visible NPC to become attracted to you.",
         "passive": null
       },
-      "flavour": "There’s only one thing I want to change about you, and that’s your last name.",
+      "flavour": "Here I am - Your other two wishes?",
       "image": "res/art/illustration/Arouse.png"
     }
   },
   {
-    "title": "Seduce",
+    "title": "Bond",
     "upgrade": null,
     "tier": "gold",
     "details": {
       "rules": {
-        "active": "Force any NPC you can name to act on their desire for you.",
+        "active": "Cause a visible NPC to bond to you for life.",
         "passive": null
       },
-      "flavour": "Here I am - Your other two wishes?",
+      "flavour": "There’s only one thing I want to change about you, and that’s your last name.",
```

Comment:
Changed this one to be more story-altering. Also fits the marriage picture better.

Swapped the flavour text of this and seduce since this one belongs on the marriage picture